examples/assemblyscript/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "wasi-hello-world",
+  "version": "1.0.0",
+  "description": "Hello world in Wasi with AS and as-wasi",
+  "main": "index.js",
+  "scripts": {
+    "build": "asc wasi-hello-world.ts -b wasi-hello-world.wasm -t wasi-hello-world.wat --runtime half",
+    "wasmtime": "wasmtime wasi-hello-world.wasm"
+  },
+  "author": "Aaron Turner",
```

Comment:
Just wanted to make sure your cool with the license and me being the author. Feel free to change to whichever you think is best for the wasmtime repo :smile:

Reply:
Thanks for the heads up! I think we can leave it as is for now.

Of course, if you'd ever like us to change it, feel free to reach out.